src/p2p/net_node.inl

```diff
@@ -1188,6 +1189,7 @@ namespace nodetool
     }
     else if (!just_take_peerlist)
     {
+      if (context_.support_flags == 0)
```

Comment:
The refusal to make whitespace changes seems a bit much here (and below). The behavior of these three lines is changing after all.